aten/src/ATen/core/OpsAlreadyMovedToC10.cpp

```diff
@@ -1098,6 +1098,9 @@ bool aten_op_is_not_moved_to_c10_yet(const c10::OperatorName& opName) {
         {"aten::to", "dtype_layout"},
         {"aten::to", "device"},
         {"aten::to", "dtype"},
+        {"aten::result_type", ""},
```

Comment:
why is this here 3 times?

Reply:
rather, 2 times.  oh, I guess this is related to the name overloads, which is discussed below.

Reply:
this is a generated file so I'm not entirely sure, but yeah seems to be due to overloads

Reply:
for now I'd just add name overloads using whatever pattern you find in the file (maybe use pow as an example?)